### PR TITLE
Moved to ghcr image repository

### PR DESCRIPTION
**Pull request by mrsabath**
Replace references to old image repository `tsidentity` with the new one `ghcr.io/spiffe`. 
Signed-off-by: Mariusz Sabath <mrsabath@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions 
+.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions
 
-CONTAINER_TAG ?= tsidentity/tornjak-spire-server:latest
-CONTAINER_VERSION_IMAGEPATH ?= tsidentity/tornjak-spire-server
-CONTAINER_MANAGER_TAG ?= tsidentity/tornjak-manager:latest
+CONTAINER_TAG ?= ghcr.io/spiffe/tornjak-spire-server:latest
+CONTAINER_VERSION_IMAGEPATH ?= ghcr.io/spiffe/tornjak-spire-server
+CONTAINER_MANAGER_TAG ?= tghcr.io/spiffe/tornjak-manager:latest
```
Comment (lumjjb) [inline Makefile:5]: typo `tghcr.io`

File: docs/spire-quickstart.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ We want to replace the image with the Tornjak image: `tsidentity/tornjak-spire-s
       serviceAccountName: spire-server
       containers:
         - name: spire-server
-          image: tsidentity/tornjak-spire-server:1.0.0
+          image: ghcr.io/spiffe/tornjak-spire-server:1.0.1
```
Comment (lumjjb) [inline docs/spire-quickstart.md:171]: quick start uses 1.0.0 (https://github.com/spiffe/spire-tutorials/blob/master/k8s/quickstart/server-statefulset.yaml#L23) for now so let's stick to 1.0.0

Comment (mrsabath) [reply docs/spire-quickstart.md]: Fixed

File: Makefile (hunk 1 above)
Comment (mrsabath) [reply Makefile]: Removed the changes

**Review by lumjjb — APPROVED** [review]
LGTM

**Pull request merged**